plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -73,3 +73,60 @@ Compressor.Threshold="Threshold (dB)"
 Compressor.AttackTime="Attack (ms)"
 Compressor.ReleaseTime="Release (ms)"
 Compressor.OutputGain="Output Gain (dB)"
+Sepia.Tone="Sepia Tone"
```

Comment:
Could you provide an example?  I've only ever seen dots/periods used for grouping in the OBS lang files.  The [lang strings for the compressor filter](https://github.com/c3r1c3/obs-studio/blob/e3a9e4ede11691d2fb42e586e4bc64441d55894c/plugins/obs-filters/data/locale/en-US.ini#L70) are a simple example.